### ISSUE TITLE
Web: Link to KEGG broken 

### ISSUE THREAD

**Issue by sneumann** [ipb-halle/MetFragRelaunched#87]
Hi, 
as reported in https://github.com/ipb-halle/MetFrag/issues/22#issuecomment-1049215744,
in the web result list the KEGG link shows an additional `=` at the end:
http://www.kegg.jp/dbget-bin/www_bget?cpd:C18858=

Online KEGG Query is executed in 
https://github.com/ipb-halle/MetFragRelaunched/blob/a61900b82c0cd5835f0b7d6ae822054b1e3eb104/MetFragLib/src/main/java/de/ipbhalle/metfraglib/database/OnlineKeggDatabase.java#L38

and makes a query to e.g. http://rest.kegg.jp/find/compound/253.9-254.0/exact_mass
which gives the correct `cpd:C18858	253.966126`. 
The downloaded candidates in MetFragWeb_Candidates.xls have the correct 
```
Identifier
C18858
```

The link is created in (note: switch to https soon!)
https://github.com/ipb-halle/MetFragRelaunched/blob/a61900b82c0cd5835f0b7d6ae822054b1e3eb104/MetFragWeb/src/main/java/de/ipbhalle/metfragweb/datatype/Molecule.java#L204

with a sanitized (?) identifier:
https://github.com/ipb-halle/MetFragRelaunched/blob/a61900b82c0cd5835f0b7d6ae822054b1e3eb104/MetFragWeb/src/main/java/de/ipbhalle/metfragweb/datatype/Molecule.java#L131
and rendered in 
https://github.com/ipb-halle/MetFragRelaunched/blob/a61900b82c0cd5835f0b7d6ae822054b1e3eb104/MetFragWeb/src/main/webapp/includes/resultTable.xhtml#L174

My guess: the `p:link` has changed behavior. 
I don't think `getDatabaseLink()` adds the spurious `=`, but no guarantee 

Yours,
Steffen

**Comment by meier-rene** [comment]
I hope I found the culprit. Hopefully fixed with b006ea964ec7e49021d3d98517f2599838fb4f4b.

**Comment by meier-rene** [comment]
Ahh, now I understand the issues. I commented here in the wrong issue. Should have gone to https://github.com/ipb-halle/MetFrag/issues/22. My commit does not fix this link issue.

**Comment by meier-rene** [comment]
Ok, this is related to the dependency:
```
<dependency>
<groupId>org.primefaces</groupId>
<artifactId>primefaces</artifactId>
</dependency>

```
Version 6.2 is fine. Need to check whats going on there.

**Comment by meier-rene** [comment]
"Fixed" somehow with 9fb6813c6674c67e540662fcd9020b8296b4e3a6 by reverting back to primefaces 6.2. We need to stick with 6.2 or dig way deeper into this issue. Without urgent need I would like to let this dependency at the old version.
